integration-tests/bats/replication.bats

```diff
@@ -99,3 +99,46 @@ teardown() {
     [ "${#lines[@]}" -eq 2 ]
     [[ "$output" =~ "t1" ]] || false
 }
+
+@test "replication: no remote error" {
+    cd repo1
+    dolt config --local --add sqlserver.global.DOLT_REPLICATE_TO_REMOTE unknown
+    run dolt sql -q "create table t1 (a int primary key)"
+    [ "$status" -eq 1 ]
+    [[ ! "$output" =~ "panic" ]] || false
+    [[ "$output" =~ "failure loading hook; remote not found: 'unknown'" ]] || false
+}
+
+@test "replication: quiet replication warnings" {
+    cd repo1
+    dolt config --local --add sqlserver.global.dolt_skip_replication_errors true
+    dolt config --local --add sqlserver.global.DOLT_REPLICATE_TO_REMOTE unknown
```

Comment:
Why are we using different cases for these two?

SQL system vars are case insensitive so it shouldn't matter, but we should pick a convention and stick to it